python/pyspark/errors/error_classes.py

```diff
@@ -79,11 +114,26 @@
       "returnType can not be specified when `<arg_name>` is a user-defined function, but got <return_type>."
     ]
   },
+  "CANNOT_UNPERSIST_BROADCAST": {
+    "message": [
+      "Broadcast can only be unpersisted in driver."
+    ]
+  },
   "COLUMN_IN_LIST": {
     "message": [
       "`<func_name>` does not allow a Column in a list."
     ]
   },
+  "CONTEXT_ONLY_VALID_ON_DRIVER" : {
+    "message" : [
+      "It appears that you are attempting to reference SparkContext from a broadcast variable, action, or transformation. SparkContext can only be used on the driver, not in code that it run on workers. For more information, see SPARK-5063."
+    ]
+  },
+  "CONTEXT_UNAVAILABLE_FOR_REMOTE_CLIENT" : {
+    "message" : [
+      "Remote client cannot create a SparkContext. Create SparkSession instead."
```

Comment:
I think the message is misleading, it implies a `SparkSession` was created.

Reply:
Sorry, I didn't catch the point. The message suggests using a Spark session because it is not possible to create a SparkContext in a remote session. Do you have any proposed suggestion?

Reply:
NVM, I feel it says that `Remote client cannot create a SparkContext.  So the SparkConnect Python Client create a session instead`, while it actually raise a error and fails.
Not a big deal.